ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -23,6 +23,7 @@ StatusMenu {
     property string myPublicKey: ""
     property bool amIChatAdmin: false
     property bool disabledForChat: false
+    property bool showOnlyEmojiReactions: false
```

Comment:
"show only" means that as soon as we set this to `true`, no other options will appear.
But that wouldn't be true when `disabledForChat` is false.

```suggestion
    property bool forceEnableEmojiReactions: false
```